src/alignment.js

```diff
@@ -85,8 +85,35 @@ var alignWithDOM = function(nodeName, key, statics) {
 };
 
 
+/**
+ * Clears out any unvisited Nodes, as the corresponding virtual element
+ * functions were never called for them.
+ * @param {!Element} node
+ */
+var clearUnvisitedDOM = function(node) {
+  var data = getData(node);
+  var lastChild = node.lastChild;
```

Comment:
No point in keep track of this as a separate var. If there were DOM nodes removed / added, the amount of time that takes will be huge compared to saving 1 lookup.